pkg/rc/farm.go

```diff
@@ -228,7 +229,7 @@ START_LOOP:
 				go func(id fields.ID) {
 					defer func() {
 						if r := recover(); r != nil {
-							err := util.Errorf("Caught panic in rc farm: %s", r)
+							err := util.Errorf("Caught panic in rc farm: %s\n%s", r, debug.Stack())
 							rcLogger.WithError(err).
```

Comment:
I do like the "caught panic" message because it shows me that the panic was swallowed and didn't crash the server. It is slightly unfortunate that the line number doesn't point to the _actual_ problem, but since that information is present in the stack trace I think it's probably okay?